src/components/Sections/hooks/useShortcutsDirectories.ts

```diff
@@ -0,0 +1,32 @@
+import { useQuery } from 'cozy-client'
+import { mkHomeCustomShorcutsDirConn, mkHomeCustomShorcutsConn } from 'queries'
+import { formatShortcuts } from 'components/Shortcuts/utils'
+import { Section } from 'components/Sections/SectionsTypes'
+
+export const useShortcutsDirectories = (
+  magicHomeFolderId: string | undefined
```

Comment:
you can put an declare it undefined with `?` to make it more concise  
`magicHomeFolderId?: string`